### PR TITLE
Byobu startup script updates

### PR DESCRIPTION
**Pull request by fergusL**
PR contains a few improvements that were suggested by @wtgee 
-added a `call_byobu()` function to replace the repeated `subprocess.call()` calls/make script easier to maintain
-fixed the problem that was leading to the creation of an extra unnamed byobu window (the `new_window()` function creates a new window and then renames it, so needed a function to just rename the current window and use that on the starting window before creating more)
-brought in some typo fixes/general modifications from the version on the control computer

### CODE REVIEW

**Comment by fergusL** [comment]
@Physarah @JAAlvarado-Montes if there is anything that you think should be added to the byobu windows could you let me know so I can add/update the current setup?

File: scripts/byobu_startup.py [hunk 1]
```diff
@@ -38,13 +38,35 @@
...
+    shell : bool
+        This argument sets whether to run the subprocess call cmd through a
+        shell, as opposed treating the input command as the name of an
+        executable to be run.
+    executable : sh
+        The path to the shell executable you wish to use.
+
+    """
+    run_cmd = f'{screen_cmd} {cmd}'
+    subprocess.call(run_cmd, shell=shell, executable='/bin/bash')
```
Comment (wtgee) [inline scripts/byobu_startup.py]:
```suggestion
    subprocess.call(run_cmd, shell=shell, executable=executable)
```

File: scripts/byobu_startup.py [hunk 2]
```diff
@@ -172,37 +209,37 @@ def setup_main_control_window(cmd_prefix='#'):
...
     send_command_to_pane(
-        cmd_prefix + '$HUNTSMAN_POCS/scripts/pyro_name_server.py', 0)
+        cmd_prefix + '${HUNTSMAN_POCS}/scripts/pyro_name_server.py', 0)
 
     send_command_to_pane(
-        cmd_prefix + 'python $HUNTSMAN_POCS/scripts/start_config_server.py', 1)
+        cmd_prefix + 'python ${HUNTSMAN_POCS}/scripts/start_config_server.py', 1)
 
-    send_command_to_pane(f'ipython', 2)
+    send_command_to_pane(f'jupyter-lab', 2)
```
Comment (wtgee) [inline scripts/byobu_startup.py]: I'm not sure if we've decided this is what we want actually. Might be better to stick with `ipython`.

File: scripts/byobu_startup.py [hunk 3]
```diff
@@ -172,37 +209,37 @@ def setup_main_control_window(cmd_prefix='#'):
...
-        time.sleep(0.01)
+    #for cmd in POCS_STARTUP:
+    #    send_command_to_pane(cmd_prefix + cmd, 2)
+    #    # issues occur without small pause between commands, unsure why
+    #    time.sleep(0.01)
 
-    send_command_to_pane(
-        cmd_prefix + 'grc tail -F -n 1000 $PANDIR/logs/ipython-all.log', 3)
+    #send_command_to_pane(
+    #    cmd_prefix + 'grc tail -F -n 1000 $PANDIR/logs/ipython-all.log', 3)
```
Comment (wtgee) [inline scripts/byobu_startup.py]: If this is going to stay commented let's remove. Otherwise shouldn't the "dry run" flag make it so that it skips this?

**Review by wtgee — APPROVED** [review]
LGTM, thanks! Minor suggestions added.

File: scripts/byobu_startup.py (hunk 3 above)
Comment (fergusL) [reply scripts/byobu_startup.py]: I think this was something that was changed on the control computer and I just copied it over. I'm not sure why someone chose to comment it out

**Pull request merged**